src/color.rs

```diff
@@ -1,15 +1,15 @@
-use serde::{Serialize, Deserialize};
 pub use hex_color::HexColor;
+use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub enum PresetColor {
-    Red,
-    Orange,
-    Yellow,
-    Green,
-    Cyan,
-    Purple,
+    Red = 1,
```

Comment:
What the purpose of adding some numerical values for enum?

Reply:
The spec says that the values exist but not what they're for, but I thought I'd get them in there in some form just in case they were helpful later...

This issue https://github.com/obsidianmd/jsoncanvas/issues/37#issuecomment-2016530906 has just clarified what they mean/ how they're used, and means that we'll need to update how preset colours are serialised.

Reply:
Fixed